### PR TITLE
Feature/workbox

### PR DESCRIPTION
**Pull request by danial24**
## What I did
PWA via Workbox

### CODE REVIEW

File: packages/web/src/server/components/HTML/index.tsx [hunk 1]
```diff
@@ -35,10 +35,10 @@ const HTML = (props: HTMLProperties) => {
 							content="width=device-width, initial-scale=1"
 						/>
 						<meta name="msapplication-TileColor" content="#2b2b2b" />
-						<meta
+						{/* <meta
```
Comment (artalat) [inline packages/web/src/server/components/HTML/index.tsx]: if we dont need this, can we remove this code?

Comment (danial24) [reply packages/web/src/server/components/HTML/index.tsx]: Yes will remove it after going through the usage of this file in detail.

**Comment by danial24** [comment]
@artalat 
There is only one check that is failing. Do let me know of this.
Please Review and Merge the PR.

File: packages/web/src/server/components/HTML/index.tsx (hunk 1 above)
Comment (artalat) [reply packages/web/src/server/components/HTML/index.tsx]: are we doing this?

**Comment by artalat** [comment]
@danial24 where do we stand on this PR?

**Comment by danial24** [comment]
@artalat 
The develop branch was broken catering lint.
That is why it was failing.

There was no lint script in package.json in the expo package. Added lint and fix command there.
Fixed linting issues as well.
The PR is ready to merge

**Comment by artalat** [comment]
@danial24 does this also resolve #48?

**Comment by danial24** [comment]
@artalat 
I dont think so. I guess we will have to fix it as well.

**Pull request merged**